src/pages/studyView/StudyViewPage.tsx

```diff
@@ -5,70 +5,135 @@ import $ from 'jquery';
 import {observer} from "mobx-react";
 import {remoteData} from "../../shared/api/remoteData";
 import {action, computed, observable, reaction} from "mobx";
-import client from "shared/api/cbioportalInternalClientInstance";
+import internalClient from "shared/api/cbioportalInternalClientInstance";
+import defaultClient from "shared/api/cbioportalClientInstance";
+import { ClinicalDataCountFilter, StudyViewFilter, ClinicalDataCount, ClinicalDataEqualityFilter } from 'shared/api/generated/CBioPortalAPIInternal';
+import { PieChart } from './charts/pieChart/PieChart'
+import { ClinicalAttribute } from 'shared/api/generated/CBioPortalAPI';
+import styles from "./styles.module.scss";
 
 
-function getMutatedGenes(query:string){
-    console.log("fetching genes");
-    return new Promise((resolve,reject)=>{
-        setTimeout(function(){
-            resolve(_.filter(["aaron","nora","natasha", "john"],(name:string)=>name.includes(query)))
-        },1000);
-    });
-
-}
-
-
-// this is an example of using client.
-client.getAllGenesetsUsingGET({}).then(function(data){
-    console.log(data);
-});
-
-class StudyViewPageStore {
+export type ClinicalAttributeData = {[attrId:string]:ClinicalDataCount[]};
+export type ClinicalAttributeDataWithMeta = {attributeId:string,clinicalDataType:"SAMPLE" | "PATIENT",counts:ClinicalDataCount[]};
+export class StudyViewPageStore {
 
     constructor(){
 
     }
 
-    @observable filterTerm = "";
-
-    readonly mutatedGenes = remoteData({
-
-        await:()=>[
-            this.prelimData
-        ],
-
-        invoke:()=>{
-
-            return getMutatedGenes(this.filterTerm);
-
+    @observable studyId = "hnsc_tcga"
+
+    @observable sampleAttrIds = ["OCT_EMBEDDED"]
+
+    @observable patientAttrIds = ["AJCC_PATHOLOGIC_TUMOR_STAGE","ICD_O_3_HISTOLOGY"]
+    
+    @observable filter:StudyViewFilter;
+
+    @observable propFilters:{[id:string]:ClinicalDataEqualityFilter} = {}
```

Comment:
can we call this clinicalDataFilters?   "props" in React should be reserved for React props